base/doc/ltnews40.tex

```diff
@@ -145,6 +145,15 @@ \section{Introduction}
 
 % To write
 
+\section{Switch to T1 as default encoding in documents using \cs{DocumentMetadata}}
+
+As it is well known the font encoding OT1 supports only 128 characters and has various problems and quirks notably for languages different to english. Nevertheless OT1 is the default encoding in \LaTeX{} and this can not be easily changed without affecting many documents as the T1 version of the fonts have slightly different metrics. 
+
+The introduction of the \cs{DocumentMetadata} command, which announces \emph{new} code
+and changes that can also affect the layout gives us now the opportunity to make this step. So with this version a use of \cs{DocumentMetadata} with (pdf)\LaTeX{} will setup  T1 as default font encoding\footnote{The unicode engines will continue to use TU as encoding.}. To ensure that scalable fonts are used, the package \pkg{cm-super} has to be installed. User that want to revert to the OT1 encoding in their document can do this \verb+\usepackage[OT1]{fontenc}+.
```

Comment:
'Unicode'

Reply:
also 'User that want to ...' -> 'Users ...'

```suggestion
and changes that can also affect the layout gives us now the opportunity to make this step. So with this version a use of \cs{DocumentMetadata} with (pdf)\LaTeX{} will setup  T1 as default font encoding\footnote{The Unicode engines will continue to use TU as encoding.}. To ensure that scalable fonts are used, the package \pkg{cm-super} has to be installed. Users that want to revert to the OT1 encoding in their document can do this \verb+\usepackage[OT1]{fontenc}+.
```

Reply:
Better: . . . Users who . . .